harness/determined/cli/experiment.py

```diff
@@ -1185,6 +1199,21 @@ def experiment_id_arg(help: str) -> Arg:  # noqa: A002
                 ),
             ],
         ),
+        # Continue experiment command.
+        Cmd(
+            "restore",
```

Comment:
it's confusing that the REST API is called `ContinueExperiment`, while the CLI is `restore`. I'd highly recommend renaming one of them. If `restore` is the desired user-facing naming, can we rename the REST API the same way?

then the `continue_hp_search` function also could be called `restore_experiment`; Python SDK also should be called the same as CLI, make it all uniform.

Reply:
The idea behind this was ```continue``` is more feature packed, allowing you to do things like change the config value and train you experiment for a longer period of time. Right now only single searcher experiments can change the config due to it being a lot more difficult to implement that for hp search. 

I'm not against just making the CLI be one continue with documenting that non single search experiments can't provide a config. I think that might be a better call here just to avoid introducing more complexity to the user.

Reply:
for a user, it'd probably be hard to tell `continue` and `restore` verbs apart. if there are limitations depending on the searcher type, it'd probably be simpler to report these as an error message, as early as possible?